apps/native-component-list/src/screens/FaceDetector.tsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import { Image, Platform, View, ScrollView } from 'react-native';
+import { ImagePicker, FaceDetector, Permissions } from 'expo';
+
+import ListButton from '../components/ListButton';
+import MonoText from '../components/MonoText';
+import { FaceFeature } from '../../../../packages/expo-face-detector/build/FaceDetector';
```

Comment:
Well, that seems not to be correct way of importing this interface/type. Please expose it via `expo` package, that means via `expo-face-detector` package